Octokit/Models/Response/Readme.cs

```diff
@@ -30,6 +30,22 @@ internal Readme(ReadmeResponse response, IApiConnection client)
             htmlContent = new Lazy<Task<string>>(async () => await client.GetHtml(Url).ConfigureAwait(false));
         }
 
+        internal Readme(ReadmeResponse response, IConnection client)
```

Comment:
Yep, you're right! I've missed opportunity to use substitution of IApiConnection instead of new Readme constructor. Fixed :+1: